src/model/controllers/modelController.ts

```diff
@@ -7,8 +7,10 @@ import { SERVICES } from '../../common/constants';
 import { IngestionPayload } from '../../common/interfaces';
 import { ModelManager } from '../models/modelManager';
 import { StoreTriggerResponse } from '../../externalServices/storeTrigger/interfaces';
+import { MetadataParams } from '../../externalServices/catalog/interfaces';
```

Comment:
Move the MetadataParams outside the externalServices. Put it in interface inside common

Reply:
ok